File: configlibrary/src/main/java/com/shmedo/configlibrary/iot/model/das/DasNetStatusInfo.java
package com.shmedo.configlibrary.iot.model.das;

/**
 * 创建者:   gonghe <br/>
 * 创建时间:  2021/4/15 <br/>
 * 描述：      DAS 状态页面数据中心状态
 */
public class DasNetStatusInfo {

    private int index;//中心编号
    private int errno;//错误码
    private int send;//已发送
    private int unsend;//未发生
    private float rate;//在线率

    public int getIndex() {
        return index;
    }

    public void setIndex(int index) {
        this.index = index;
    }

    public int getErrno() {
        return errno;
    }

    public void setErrno(int errno) {
        this.errno = errno;
    }

    public int getSend() {
        return send;
    }

    public void setSend(int send) {
        this.send = send;
    }

    public int getUnsend() {
        return unsend;
    }

    public void setUnsend(int unsend) {
        this.unsend = unsend;
    }

    public float getRate() {
        return rate;
    }

    public void setRate(float rate) {
        this.rate = rate;
    }
}

File: lib_cmd/src/main/java/com/shmedo/lib/cmd/base/iot_cmd/model/das/DasNetStatusInfo.kt
package com.shmedo.lib.cmd.base.iot_cmd.model.das

import com.squareup.moshi.JsonClass

/**
 * 创建者:   gonghe <br></br>
 * 创建时间:  2021/4/15 <br></br>
 * 描述：      DAS 状态页面数据链路状态
 */
@JsonClass(generateAdapter = true)
data class DasNetStatusInfo(
    var index: Int = 0, //中心编号
    var errno: Int = 0, //错误码
    var send: Int = 0,//已发送
    var unsend: Int = 0, //未发送
    var rate: Float = 0f,//在线率
)